subworkflows/local/eggnog.nf

```diff
@@ -15,7 +15,6 @@ workflow EGGNOG {
     ch_versions = Channel.empty()
 
     EGGNOG_DOWNLOAD()
-    ch_versions = ch_versions.mix ( EGGNOG_DOWNLOAD.out.versions )
```

Comment:
so if I understand correctly, we don't have version.yml for both EGGNOG_DOWNLOAD and KOFAMSCAN_DOWNLOAD module, right?

Reply:
Correct. I don't know what difference it makes for the download (it might stop a few, as it did for me) and versions for this are either just `wget` or a tool that gets reported anyway.